client_admin/pulp/client/admin/tasks.py

```diff
@@ -36,20 +41,29 @@ class BaseTasksSection(PulpCliSection):
     state_option = PulpCliOption('--state',
                                  _('comma-separated list of tasks states desired to be '
                                    'shown. Example: "running,waiting,canceled,successful,failed". '
-                                   'Do not include spaces'), aliases=['-s'], required=False,
-                                 parse_func=parsers.csv)
+                                   'Do not include spaces.'),
+                                 aliases=['-s'], required=False, parse_func=parsers.csv)
+    # Flags for purge command
+    purge_all = PulpCliFlag('--all', _('if specified, all tasks in "finished, '
+                                       'skipped and error" states can be purged'),
+                            aliases=['-a'])
+    purge_state = PulpCliOption('--state', _('comma-separated list of tasks states desired to be '
+                                'purged. Example: "finished,error". Do not include spaces.'),
```

Comment:
@shubham90 I think you should do the same as i did here https://github.com/shubham90/pulp/blob/i228/client_admin/pulp/client/admin/tasks.py#L315
As when a user list tasks, he sees tasks in state: successful and not finished, failed and not error. We have this discrepancy between cli and api but the user should not be confused by this. Please also update docs and code throughout the PR
```
$ pulp-admin tasks list --all

+----------------------------------------------------------------------+
                                 Tasks
+----------------------------------------------------------------------+

Operations:  delete
Resources:   test (repository)
State:       Successful
Start Time:  2015-07-27T09:58:07Z
Finish Time: 2015-07-27T09:58:07Z
Task Id:     b674bc1d-318a-425c-b337-565adf198658

[ipanova@ina pulp]$ pulp-admin tasks purge --state successful


+----------------------------------------------------------------------+
                         Purge Completed Tasks
+----------------------------------------------------------------------+

Invalid task state passed to purge: successful. Valid states are finished, error
and/or skipped